src/zoid/installments/component.jsx

```diff
@@ -33,14 +33,16 @@ export function getInstallmentsComponent() : InstallmentsComponent {
             logger: getLogger(),
 
             prerenderTemplate: ({ doc, props }) => {
+                const nonce = props.nonce || getCSPNonce();
```

Comment:
Why is the or condition needed?

```suggestion
                const { nonce } = props;
```